README.md

```diff
@@ -19,21 +19,14 @@ configuration that expedites the setup process.
 For Windows machines, a convenient installation tool is provided that installs
 all the necessary software for you. See [this article](https://www.raspberrypi.com/news/raspberry-pi-pico-windows-installer/) to download the installer.
 
-#### MacOS
+#### macOS
 
-For MacOS, a manual installation is required. Please make sure that Homebrew, a
-package manager for MacOS, is installed. If not, you can install it by running
-
-```shell
-$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-```
-
-Then, run the following commands to install the rest of the required software:
+For macOS, CMake and the ARM compiler for the Raspberry Pi Pico can be installed from
+[Homebrew formulae](https://brew.sh).
 
 ```shell
 $ brew install cmake
-$ brew tap ArmMbed/homebrew-formulae
-$ brew install arm-none-eabi-gcc
+$ brew install gcc-arm-embedded
```

Comment:
Question: What's the difference between these two formulae?

Reply:
As noted in [ARMmbed/homebrew-formulae](https://github.com/ARMmbed/homebrew-formulae), the older `arm-none-eabi-gcc` is deprecated and replaced with `gcc-arm-embedded` as also suggested in raspberrypi/pico-feedback#355.